### PR TITLE
Release notes for Kots 1.12.0

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/release-notes/1.12.0.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
+* Airgap upload doesn't redirect back to license upload.
+* A failed airgap upload will display an error.
+* UI reflects failing preflight checks correctly.
+* Fixed an issue where Preflight checks fail if a `when` condition is missing.
+* Fixed issues with kURL HA registry availability when a node is lost.
+* Fixed issues with assigning customers to different channels.
```
Comment (replicated-collab) [inline content/release-notes/1.12.0.md]: What was the issue that occurred when assigning customers to different channels? How did this bug manifest?

Comment (austinchambers) [inline content/release-notes/1.12.0.md]: What was the issue that occurred when assigning customers to different channels? How did this bug manifest?

File: content/release-notes/1.12.0.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
+* Airgap upload doesn't redirect back to license upload.
+* A failed airgap upload will display an error.
+* UI reflects failing preflight checks correctly.
+* Fixed an issue where Preflight checks fail if a `when` condition is missing.
+* Fixed issues with kURL HA registry availability when a node is lost.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: Assuming the issue here is the registry is not running when a node is lost?

File: content/release-notes/1.12.0.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
+* Airgap upload doesn't redirect back to license upload.
+* A failed airgap upload will display an error.
+* UI reflects failing preflight checks correctly.
+* Fixed an issue where Preflight checks fail if a `when` condition is missing.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: Not sure I understand. If a when condition is missing, then there's no condition on the preflight?

File: content/release-notes/1.12.0.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
+* Airgap upload doesn't redirect back to license upload.
+* A failed airgap upload will display an error.
+* UI reflects failing preflight checks correctly.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: How did the UI improperly reflect failing preflight checks?

File: content/release-notes/1.12.0.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* Helm chart rendering errors are now displayed in the Admin Console.
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
+* Airgap upload doesn't redirect back to license upload.
+* A failed airgap upload will display an error.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: To clarify, we were previously not showing an error, and now we do, right?

File: content/release-notes/1.12.0.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
+* Airgap upload endpoints now require `Authorization` headers.
+* Helm chart rendering errors are now displayed in the Admin Console.
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
+{{</changes>}}
+
+{{<fixes>}}
+* Airgap upload no longer tries to authenticate with docker registry.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: Why not? What happened when it tried to authenticate with docker registry?

File: content/release-notes/1.12.0.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+* If license is updated during a sync, a dialog is shown linking to version history.
+* Better feedback when saving app config.
+* Support bundle analysis now has breadcrumbs.
+* `replicated/kurl-util` container uses a versioned tag instead of `latest`.
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
+* Airgap upload endpoints now require `Authorization` headers.
+* Helm chart rendering errors are now displayed in the Admin Console.
+* The Dashboard link has been removed from the license upload pages.
+* License is synced to the latest version on installation.
+* `kotsadm-api` no longer needs to be restarted when console password is reset.
```
Comment (austinchambers) [inline content/release-notes/1.12.0.md]: Sounds like a bug fix. So when we use the CLI to reset password, it didn't take until we also restart kotsadm-api?

File: content/release-notes/1.12.0.md (hunk 6 above)
Comment (divolgin) [reply content/release-notes/1.12.0.md]: It's airgap.  There is no internet access.

Comment (austinchambers) [reply content/release-notes/1.12.0.md]: I understand, but you can access the internet in some airgap installs. I've used the docker registry in an airgap install with no problem.  Is this no longer possible?

File: content/release-notes/1.12.0.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-01-31T19:55:39-08:00
+linktitle: "1.12.0"
+title: "1.12.0"
+kubernetes: "1.15, 1.16, and 1.17"
+---
+{{<features>}}
+* Namespace can now be specified for [Helm charts in KOTS](https://kots.io/reference/template-functions/static-context/)
```
Comment (marccampbell) [inline content/release-notes/1.12.0.md]: I think this link is wrong, it should link to https://kots.io/reference/v1beta1/helmchart/#namespace

File: content/release-notes/1.12.0.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+* Preflight and Troubleshoot analyzers now support analysis of Node resources, including [CPU, Memory, and Disk Space](https://troubleshoot.sh/reference/analyzers/node-resources/)
+{{</features>}}
+
+{{<changes>}}
+* If license is updated during a sync, a dialog is shown linking to version history.
+* Better feedback when saving app config.
+* Support bundle analysis now has breadcrumbs.
+* `replicated/kurl-util` container uses a versioned tag instead of `latest`.
+* `kots` CLI now includes an `Authorization` header when making requests to `kotsadm-api`.
```
Comment (marccampbell) [inline content/release-notes/1.12.0.md]: @austinchambers can we link to docs on how to use this header in automation scripts?

**Pull request merged**